tests/code/test_serviceLearningCourses.py

```diff
@@ -38,6 +43,21 @@ def test_getServiceLearningCoursesData():
     assert "In Progress" == courseDict[4]['status']
     assert 'Spring 2021' in courseDict[4]['term'].description
     assert "Scott Heggen"  == courseDict[4]['creator']
+    assert "Testing" == courseDict[4]['courseDisplayName']
+
+
+
+@pytest.mark.integration
+def test_createCourseDisplayName():
+    '''tests for the successful implementation of combining course name and number to proper format'''
+    
+    assert 'Databases' == createCourseDisplayName("Databases", '')
+    assert 'Databases' == createCourseDisplayName("Databases", "")
+    assert 'FRN 103 - Frenchy Help' == createCourseDisplayName("Frenchy Help", 'FRN 103')
+    assert 'FRN 103' == createCourseDisplayName("", 'FRN 103')
+    assert '' == createCourseDisplayName ("", '')
```

Comment:
one extra whitespace on 58

Reply:
Thanks for the review will look into and remove the whitespace